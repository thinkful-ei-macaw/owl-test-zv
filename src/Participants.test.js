import renderer from 'react-test-renderer';

/***** Smoke test *****/
// make React available
import React from 'react';

// make the ReactDOM available, necessary for rendering the component
import ReactDOM from 'react-dom';

// make the App component available
import List from './Participants';

// this is the test case
it('renders without crashing', () => {
  // first create a DOM element to render the component into
  const div = document.createElement('div');

  // render the component, this is the actual test, if something is wrong it will fail here
  ReactDOM.render(<List />, div);

  // clean up code
  ReactDOM.unmountComponentAtNode(div);
});
/***** Snapshot test *****/


it('renders the UI as expected', () => {
  const tree = renderer
    .create(<List />)
    .toJSON();
  expect(tree).toMatchSnapshot();  
});