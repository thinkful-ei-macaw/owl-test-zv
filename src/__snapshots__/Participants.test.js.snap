// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the UI as expected 1`] = `
<section>
  <div>
    <div>
      <div
        className="avatar"
      >
        <img
          alt="user"
          src="https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1"
        />
      </div>
      <ul>
        <li>
          Koren Templeton
        </li>
        <ul>
          <div>
            <p>
              On Stage
            </p>
            <p>
              In Session
            </p>
          </div>
        </ul>
      </ul>
    </div>
    <div>
      <div
        className="avatar"
      >
        <img
          alt="user"
          src="https://robohash.org/doloredolorescupiditate.jpg?size=200x200&set=set1"
        />
      </div>
      <ul>
        <li>
          Caty Flucker
        </li>
        <ul>
          <div>
            
            <p>
              In Session
            </p>
          </div>
        </ul>
      </ul>
    </div>
    <div>
      <div
        className="avatar"
      >
        <img
          alt="user"
          src="https://robohash.org/etexercitationemassumenda.jpg?size=200x200&set=set1"
        />
      </div>
      <ul>
        <li>
          Frank Runciman
        </li>
        <ul>
          <div>
            
            <p>
              In Session
            </p>
          </div>
        </ul>
      </ul>
    </div>
    <div>
      <div
        className="avatar"
      >
        <img
          alt="user"
          src="https://robohash.org/iustodoloremqueinventore.jpg?size=200x200&set=set1"
        />
      </div>
      <ul>
        <li>
          Ashla Attwool
        </li>
        <ul>
          <div>
            <p>
              On Stage
            </p>
            <p>
              In Session
            </p>
          </div>
        </ul>
      </ul>
    </div>
    <div>
      <div
        className="avatar"
      >
        <img
          alt="user"
          src="https://robohash.org/utlaborumfugit.jpg?size=200x200&set=set1"
        />
      </div>
      <ul>
        <li>
          Axe Kubicka
        </li>
        <ul>
          <div>
            
            
          </div>
        </ul>
      </ul>
    </div>
  </div>
</section>
`;
